examples/tfengine/modules/team.hcl

```diff
@@ -505,6 +525,7 @@ data "google_container_cluster" "gke_cluster" {
 }
 
 provider "kubernetes" {
+  alias                  = "gke-alias"
```

Comment:
once the provider field in ksa is made optional, this can be removed.

Reply:
Done.